core/src/message/handle.rs

```diff
@@ -25,7 +25,8 @@ use crate::{
 use codec::{Decode, Encode};
 use scale_info::TypeInfo;
 
-/// Handle message.
+/// Message for Handle entry point.
+/// User send this messages to programs or another users
```

Comment:
Sentences in doc comments end with a dot.
```suggestion
/// User send these messages to programs or other users.
```

Reply:
done